test/config/testdata/artman_pubsub.yaml

```diff
@@ -3,15 +3,15 @@ common:
   api_version: v1
   organization_name: google-cloud
   src_proto_paths:
-  - google/pubsub/v1
-  - -google/pubsub/v1/excluded
+  - /tmp/input/google/pubsub/v1
+  - -/tmp/input/google/pubsub/v1/excluded
```

Comment:
What is the deal with this? Also, `- -` syntax in YAML scares me. Is that a string with a minus at the beginning, or a nested list? Will different YAML parsers even all do the same thing?

Reply:
It is a string with a minus at the beginning. It is a syntax to exclude certain file or folder.